frontend/views/containers/contributions/ContributionsWidget.vue

```diff
@@ -0,0 +1,184 @@
+<template lang='pug'>
+  page-section(:title='L("Contributions")')
+    .c-widget
+      .c-column
+        .c-status(v-if='copy.payments.title')
+          h3.is-title-4.c-title {{ copy.payments.title }}
+          .has-text-1(
+            v-html='copy.payments.status'
+            @click='handlePaymentStatusClick'
+          )
+        payments-summary.c-status(v-else)
+
+        router-link.button.is-small(
+          v-if='copy.payments.ctaText'
+          to='/pay-group'
+          :class='copy.payments.ctaClass'
+        ) {{ copy.payments.ctaText }}
+      .c-column
+        h3.is-title-4.c-title(v-html='copy.monetary.title')
+        .has-text-1.c-status {{ copy.monetary.status }}
+        i18n.link(tag='button' @click='openModal("IncomeDetails")') Change
+      .c-column
+        h3.is-title-4.c-title {{ copy.nonMonetary.title }}
+        .has-text-1.c-status {{ copy.nonMonetary.status }}
+        router-link.link(
+          to='/contributions'
+        ) {{ L('See all contributions') }}
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import sbp from '~/shared/sbp.js'
+import { OPEN_MODAL } from '@utils/events.js'
+import PageSection from '@components/PageSection.vue'
+import L from '@view-utils/translations.js'
+import currencies from '@view-utils/currencies.js'
+import PaymentsSummary from '@containers/PaymentsSummary'
+
+export default {
+  name: 'ContributionsWidget',
+  components: {
+    PageSection,
+    PaymentsSummary
+  },
+  computed: {
+    ...mapGetters([
+      'groupSettings',
+      'currentGroupState',
+      'groupProfiles',
+      'ourUserIdentityContract',
+      'ourUsername',
+      'ourContributionSummary'
+    ]),
+    withCurrency () {
+      return currencies[this.groupSettings.mincomeCurrency].displayWithCurrency
+    },
+    copy () {
+      const {
+        givingMonetary,
+        givingNonMonetary,
+        receivingMonetary,
+        receivingNonMonetary
+      } = this.ourContributionSummary
+
+      const copy = {}
+
+      if (receivingMonetary) {
+        copy.payments = {
+          title: !receivingMonetary.total && L('Payments received'),
+          status: !receivingMonetary.total && L('No members in the group are pledging yet! 😔'),
+          ctaText: receivingMonetary.total && L('See more')
+        }
+
+        copy.monetary = {
+          title: L('You need {br_}{amount}', {
+            ...this.LTags(),
+            amount: this.withCurrency(receivingMonetary.needed)
+          }),
+          status: L('You will receive {amount}.', {
+            amount: this.withCurrency(receivingMonetary.total)
+          })
+        }
+      } else if (givingMonetary) {
+        const copyMonetaryTitle = (amount) => L('You are pledging {br_} {amount}', {
```

Comment:
Should the space after `{br_}` be there?

Reply:
It's irrelevant... it has no influence on the layout. I would keep the space before and after fo readability.